addon/templates/components/polaris-empty-search-result.hbs

```diff
@@ -0,0 +1,25 @@
+{{#polaris-stack
+  alignment="center"
+  vertical=true
+}}
+  {{#if withIllustration}}
+    {{!-- TODO: replace this with polaris-image when implemented. --}}
+    <img
+      src="/illustrations/empty-search.svg"
```

Comment:
Also TODO before merging, I image this path will have to be updated. Pulling this branch into a test app now to check.